lib/fs.js

```diff
@@ -1921,79 +1921,6 @@ WriteStream.prototype.close = ReadStream.prototype.close;
 // There is no shutdown() for files.
 WriteStream.prototype.destroySoon = WriteStream.prototype.end;
 
-
-// SyncWriteStream is internal. DO NOT USE.
-// Temporary hack for process.stdout and process.stderr when piped to files.
```

Comment:
Temporary, eh?

Ideally this would be piped though a `net.Socket` so that the `_handle` type would be the same but I don't know if that is feasible.